src/win32/wazuh-installer.nsi

```diff
@@ -46,7 +46,7 @@ VIAddVersionKey ProductVersion "${VERSION}"
 VIAddVersionKey InternalName "Wazuh Agent"
 VIAddVersionKey OriginalFilename "${OutFile}"
 
-InstallDir "$PROGRAMFILES\ossec-agent"
+InstallDir "$PROGRAMFILES\wazuh-agent"
```

Comment:
We shouldn't change the installation directory. This will be done in another epic.